cmd/skipper/main.go

```diff
@@ -90,6 +108,14 @@ func init() {
 	flag.StringVar(&innkeeperPreRouteFilters, "innkeeper-pre-route-filters", "", innkeeperPreRouteFiltersUsage)
 	flag.StringVar(&innkeeperPostRouteFilters, "innkeeper-post-route-filters", "", innkeeperPostRouteFiltersUsage)
 	flag.BoolVar(&devMode, "dev-mode", false, devModeUsage)
+	flag.StringVar(&metricsListener, "metrics-listener", defaultMetricsListener, metricsListenerUsage)
+	flag.StringVar(&metricsPrefix, "metrics-prefix", defaultMetricsPrefix, metricsPrefixUsage)
+	flag.BoolVar(&debugGcMetrics, "debug-gc-metrics", false, debugGcMetricsUsage)
+	flag.BoolVar(&runtimeMetrics, "runtime-metrics", true, runtimeMetricsUsage)
+	flag.StringVar(&applicationLog, "application-log", "", applicationLogUsage)
+	flag.StringVar(&applicationLogPrefix, "application-log-prefix", "", applicationLogPrefixUsage)
+	flag.StringVar(&accessLog, "access-log", "", accessLogUsage)
```

Comment:
why are some defaults extracted as named constants and some defaults are just used inline? Seems inconsistent